README.md

```diff
@@ -58,3 +58,35 @@ To view the apache logs, run the following commands (inside Vagrant):
 $ tail -f /var/log/apache2/sylius_error.log
 $ tail -f /var/log/apache2/sylius_access.log
 ```
+
+# Configure vagrant to contribute on Sylius Core
+
+### Use sylius for contribution
+
+Edit "Vagrantfile" and replace this following line:
+
+```bash
+sylius_config.vm.provision :shell, privileged: false, path: "shell_provisioner/module/sylius.sh" 
+```
+
+By this one:
+```bash
+sylius_config.vm.provision :shell, privileged: false, path: "shell_provisioner/module/sylius_for_contribution.sh"
+```
+
+### Clone your own sylius repository
+
+```bash
+cd /sites/
+git clone https://github.com/{user}/Sylius sylius
```

Comment:
Instead of 
```
cd /sites/
git clone https://github.com/{user}/Sylius sylius
```
We could go with
```
git clone https://github.com/{user}/Sylius sites/sylius`
```
WDYT?

Reply:
yep, nice, easier is always better !